Microsoft.Toolkit.Uwp.UI.Controls/AdaptiveGridView/AdaptiveGridView.cs

```diff
@@ -49,16 +50,6 @@ public AdaptiveGridView()
             Items.VectorChanged += ItemsOnVectorChanged;
             Loaded += OnLoaded;
             Unloaded += OnUnloaded;
-
-            // Define ItemContainerStyle in code rather than using the DefaultStyle
-            // to avoid having to define the entire style of a GridView. This can still
-            // be set by the enduser to values of their chosing
-            var style = new Style(typeof(GridViewItem));
-            style.Setters.Add(new Setter(HorizontalContentAlignmentProperty, HorizontalAlignment.Stretch));
-            style.Setters.Add(new Setter(VerticalContentAlignmentProperty, VerticalAlignment.Stretch));
-            style.Setters.Add(new Setter(MarginProperty, new Thickness(0, 0, 0, 4)));
-            style.Setters.Add(new Setter(PaddingProperty, new Thickness(2, 0, 2, 0)));
```

Comment:
The `DefaultItemMargin` below was `0,0,4,4`, should it be updated to be `2,0,2,4` to match this or was that intentional?

If it's intentional that's fine, I think the buffer at the right/bottom is better than the split, but just wanted to make sure.

Reply:
The default margin for an item is 0,0,4,4. The default padding is 0. I think @nmetulev changed to this to help with knowing the width of the item